### ISSUE TITLE
Only complete directories on `add`/`dev`?

### ISSUE THREAD

**Issue by 00vareladavid** [JuliaLang/Pkg.jl#736]
`pkg> add RE` will suggest "README.md". We could even take it a step further and check that the directories have a project file before suggesting

**Comment by KristofferC** [comment]
We should never complete a `file` right?

**Comment by 00vareladavid** [comment]
Yeah, I don't think we should. I double checked: latest stable release does not suggest "README.md", but `master` does.

...but its a simple fix :grin:

**Comment by StefanKarpinski** [comment]
It might make sense to allow people to add `Project.toml` and `JuliaProject.toml` files.
